static/js/mediastream/api.js

```diff
@@ -99,11 +105,29 @@ define(['jquery', 'underscore', 'ua-parser'], function($, _, uaparser) {
 					cb(type, data);
 				}
 				this.send(type, data);
+			}, this),
+			sendEncrypted: _.bind(function(type, data) {
+				if (cb) {
+					cb(type, data);
+				}
+				this.sendEncrypted(type, data);
 			}, this)
 		}
 		return this.apply(name, obj);
 	};
 
+	Api.prototype.sendEncrypted = function(type, data, noqueue) {
```

Comment:
Yes. Stuff whichout a To field - maybe this can be bound to the document type beeing sent?